scripts/build/TestPlatform.Dependencies.props

```diff
@@ -10,8 +10,8 @@
     <MSTestAdapterVersion>1.2.0-beta</MSTestAdapterVersion>
     <MSTestAssertExtensionVersion>1.0.3-preview</MSTestAssertExtensionVersion>
     
-    <XUnitFrameworkVersion>2.2.0</XUnitFrameworkVersion>
-    <XUnitAdapterVersion>2.2.0</XUnitAdapterVersion>
+    <XUnitFrameworkVersion>2.3.0-beta3-build3705</XUnitFrameworkVersion>
+    <XUnitAdapterVersion>2.3.0-beta3-build3705</XUnitAdapterVersion>
 
     <NUnit3FrameworkVersion>3.6.1</NUnit3FrameworkVersion>
     <NUnitFrameworkVersion>2.6.4</NUnitFrameworkVersion>
```

Comment:
Due to `XUPerfTestProject.csproj` file encoded as Unicode, Change the encoding to UTF-8.  It didn't get merge properly, service still guid exists. Change to net452 at DebugType also.